FunctionalTableData/CellStyle.swift

```diff
@@ -163,10 +204,10 @@ extension CellStyle: Equatable {
 		equality = equality && lhs.accessoryType == rhs.accessoryType
 		equality = equality && lhs.selectionColor == rhs.selectionColor
 		equality = equality && lhs.backgroundColor == rhs.backgroundColor
-		equality = equality && lhs.backgroundView == rhs.backgroundView
 		equality = equality && lhs.tintColor == rhs.tintColor
 		equality = equality && lhs.layoutMargins == rhs.layoutMargins
 		equality = equality && lhs.cornerRadius == rhs.cornerRadius
+		equality = equality && lhs.backgroundViewProvider?.isEqualTo(rhs.backgroundViewProvider) ?? true
```

Comment:
What if `lhs.backgroundViewProvider` is `nil` and `rhs.backgroundViewProvider` is not?

```suggestion
		equality = equality && lhs.backgroundViewProvider?.isEqualTo(rhs.backgroundViewProvider) ?? ((lhs.backgroundViewProvider == nil) == (rhs.backgroundViewProvider == nil))
```

Reply:
I think in this case it is sufficient to write the equability check as:
```
lhs.backgroundViewProvider?.isEqualTo(rhs.backgroundViewProvider) ?? (rhs.backgroundViewProvider == nil)
```